lib/services/consumers/user.service.js

```diff
@@ -160,11 +167,15 @@ function validateUpdateToUserProperties (userProperties) {
 function validateNewUserProperties (userProperties) {
   let newUserProperties = {};
 
-  if (!Object.keys(userProperties).every(key => (typeof key === 'string' && !!config.models.users.properties[key]))) {
+  if (!Object.keys(userProperties).filter(key => key !== 'username').every(key => !!config.models.users.properties[key])) {
```

Comment:
You can do it in one loop
```js
Object.keys(userProperties).every(key => key ==='username' || !!config.models.users.properties[key])
```

Reply:
filter returns array, every returns boolean. 
the code is inside if. it will require adding some another code like ....length